plugins/modules/ec2_instance_info.py

```diff
@@ -33,6 +33,11 @@
     required: false
     default: {}
     type: dict
+  uptime:
+    description:
+      - minimum running uptime in minutes of instances.  For example if uptime is 60, it would return all instances that have run more than 60 minutes.
```

Comment:
```suggestion
      - Minimum running uptime in minutes of instances.  For example if uptime is 60, it would return all instances that have run more than 60 minutes.
```

minor formatting nit based on Ansible standards.  https://docs.ansible.com/ansible/latest/dev_guide/developing_modules_documenting.html#documentation-block